compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -284,6 +284,22 @@ object Contexts {
       println("=== end context creation trace ===")
     }
 
+    /** Whether the compiler is currently doing -Xprint
```

Comment:
Thanks @odersky , I see what you mean, now it's much neater than the previous attempt 485df22.